leapmotion-examples/LeapMotionCoreAssets
# Update Leap 2.2.7 & Bugfix

**Changes:**
- Update Leap PC and Mac DLLs and Bundles to 2.2.7
- Update Mac Quick Switch build to 2.2.7
- Fix bug in VR scenes where there were missing references in alignment system.
- Fix bug where _imageList could be accessed when null in LeapImageRetreiver

**To Test:**
- Run all VR scenes in editor and confirm they exhibit proper functionality (what the scene is meant to communicate or do) and no errors are thrown in the editor: 
 - [x] 500 Blocks
 - [x]  Hand Viewer
 - [x]  Moving Demo
 - [x]  Quick Switch Demo
 - [x] Space Time Alignment
 - [x] VR Widgets 
- Confirm that Leap OVR Camera prefabs produce reasonable, working results when added to a scene:
 - [x] LeapOVRCameraRig
 - [x] LeapOVRPlayerController
 - [x] QuickSwitchCameraRig
